test/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ if(EPROSIMA_BUILD_TESTS AND IS_TOP_LEVEL AND NOT EPROSIMA_INSTALLER)
     add_subdirectory(blackbox)
     add_subdirectory(communication)
     add_subdirectory(unittest)
-
+    add_subdirectory(dds/core/xtypes)
```

Comment:
check previously if EPROSIMA_BUILD_CORE_STANDALONE is ON.

Also, the rest of the test should not be compile if EPROSIMA_BUILD_CORE_STANDALONE is ON